### ISSUE TITLE
Add TS types and example for the browser

### ISSUE THREAD

**Issue by keemor** [lujjjh/content-disposition-attachment#10]
Hi @lujjjh,

Maybe it would be good to add example for the browser and maybe TS support

```js
import { AxiosResponse } from "axios";
import { parse } from "content-disposition-attachment";

const getFilenameFromHeaders = ({ headers }: AxiosResponse<Blob>) => {
  const defaultName = "untitled";
  try {
    const { attachment, filename } = parse(headers["content-disposition"]);
    return attachment ? filename : defaultName;
  } catch (e) {
    console.error(e);
    return defaultName;
  }
};
```
It doesn't support TS for now, so create file:
*src\\@types\content-disposition-attachment\index.d.ts*

```js
declare module "content-disposition-attachment";
```

It solves https://stackoverflow.com/questions/40939380/how-to-get-file-name-from-content-disposition/

**Comment by lujjjh** [comment]
Hey @keemor,

Thank you for your example and workaround to using this library in TypeScript. I think I'd better rewrite it in TypeScript.

**Comment by lujjjh** [comment]
I've published 0.3.1 with the TypeScript declaration file and updated the getting started section in the README.

I'm closing this issue. Please feel free to reopen it if there is any further questions.
